src/Util.cpp

```diff
@@ -139,6 +141,18 @@ dir_name(nonstd::string_view path)
   return n == 0 ? "/" : path.substr(0, n);
 }
 
+nonstd::string_view
+get_extension(nonstd::string_view path)
+{
+  const char stop_at_chars[] = "./";
```

Comment:
This and the legacy version both stop at `/` - are non-cygwin Windows paths converted from `\` to `/` elsewhere or is only cygwin supported? I only see the `DIR_DELIM_CH` define used one.

Reply:
Neither non-cygwin Windows nor cygwin Windows is supported officially (yet); see #447. That said, I guess adding backslash to `stop_at_chars` for the `_WIN32` case wouldn't hurt.